### DISCUSSION TITLE
转义问题

### DISCUSSION THREAD

**Discussion by Miuzarte** [Tsuk1ko/CQ-picfinder-robot#448] [一般讨论]
四叶草这边不会对消息段进行反转义
[☘️Shamrock-山姆摇滚/&[,] 字符转义问题](https://forum.libfekit.so/d/50-zi-fu-zhuan-yi-wen-ti)
如果gocq可以直接上报符号的话要不尝试取消掉转义？
![image](https://github.com/Tsuk1ko/cq-picsearcher-bot/assets/66856838/00b33b31-5caf-4d5f-a18e-bbf3ea419f6e)

**Comment by Tsuk1ko** [comment]
按照规范应该转义，否则会有歧义，当然我感觉可以先提供一个设置项来凑合一下

**Comment by Tsuk1ko** [comment]
增加了配置 `bot.disableMessageEscape`
